.readthedocs.yml

```diff
@@ -0,0 +1,18 @@
+
+# Required
+version: 2
+
+# Build documentation in the docs/ directory with Sphinx
+sphinx:
+  configuration: docs/code-docs/source/conf.py
+  fail_on_warning: false
+
+# Optionally build your docs in additional formats such as PDF
+formats:
+  - pdf
```

Comment:
Will this actually build the docs somewhere in PDF format? interesting

Reply:
Yep! https://docs.readthedocs.io/en/stable/downloadable-documentation.html